lib/component-data/mutations.js

```diff
@@ -1,5 +1,5 @@
 import _ from 'lodash';
-import { UPDATE_COMPONENT, REVERT_COMPONENT, ADD_SCHEMA, RENDER_COMPONENT, REMOVE_COMPONENT, ADD_DEFAULT_DATA, CURRENTLY_SAVING, OPEN_ADD_COMPONENT, CLOSE_ADD_COMPONENT, CURRENTLY_RESTORING } from './mutationTypes';
+import { UPDATE_COMPONENT, REVERT_COMPONENT, ADD_SCHEMA, UPDATE_SCHEMA_PROP, RENDER_COMPONENT, REMOVE_COMPONENT, ADD_DEFAULT_DATA, CURRENTLY_SAVING, OPEN_ADD_COMPONENT, CLOSE_ADD_COMPONENT, CURRENTLY_RESTORING } from './mutationTypes';
```

Comment:
Remember to break this into multiple lines to make it more readable

Reply:
👍